### ISSUE TITLE
Training code

### ISSUE THREAD

**Issue by Shape-Kim** [SAITPublic/rascanet#2]
Hello Jaehyoung Yoo!

First of all, thank you for your great work.

Do you have a plan to release the training code?

Moreover, could you offer the training details (e.g., batch size, learning rate) in order to reproduce your results?

Thank you!
